docs/configurations.md

```diff
@@ -38,16 +38,23 @@ The following basic configurations are available for the Consensus Commit transa
+| `scalar.db.consensus_commit.coordinator.group_commit.enabled`                          | Whether or not committing the transaction state is executed in batch mode.                                                                                                                                                                                                            | `false`                                                           |
+| `scalar.db.consensus_commit.coordinator.group_commit.slot_capacity`                    | Maximum number of slots in a group for the group commit feature. A large value improves the efficiency of group commit, but may also increase latency and the likelihood of transaction conflicts.                                                                                    | `20`                                                              |
+| `scalar.db.consensus_commit.coordinator.group_commit.group_size_fix_timeout_millis`    | Timeout to fix the size of slots in a group. A large value improves the efficiency of group commit, but may also increase latency and the likelihood of transaction conflicts.                                                                                                        | `40`                                                              |
+| `scalar.db.consensus_commit.coordinator.group_commit.delayed_slot_move_timeout_millis` | Timeout to move delayed slots from a group to another isolated group to prevent the original group from being affected by delayed transactions. A large value improves the efficiency of group commit, but may also increase the latency and the likelihood of transaction conflicts. | `1200`                                                            |
+| `scalar.db.consensus_commit.coordinator.group_commit.old_group_abort_timeout_millis`   | Timeout to abort an old ongoing group. A small value reduces resource consumption through aggressive aborts, but may also increase the likelihood of unnecessary aborts for long-running transactions.                                                                                | `60000`                                                           |
+| `scalar.db.consensus_commit.coordinator.group_commit.timeout_check_interval_millis`    | Interval for checking the group commit–related timeouts.                                                                                                                                                                                                                              | `20`                                                              |
```

Comment:
Just a thought: I suppose finding the right configuration to achieve great performance depends on many factors and requires a lot of testing and monitoring. How do you estimate the degree of difficulty for customers to acquire this know-how?

Reply:
Really good question... As you thought, many factors including underlying database spec and access pattern by the applications are related. I think it would be worth trying 2 or 3 values per configuration except for `old_group_abort_timeout_millis` and `timeout_check_interval_millis`. In the case of `slot_capacity`, trying `20`, `40`, `60` may be good to find the best configuration. But the combination of these configurations is also important. So, the number of attempts would be 3 (for `slot_capacity`) times 2 (for `group_size_fix_timeout_millis`) times 2 (for `delayed_slot_move_timeout_millis)`, then 12. It might be a bit tough for users, though.

Reply:
I see. Thanks for laying out a practical approach to get closer to a good configuration. Adding that to the documentation would already be quite helpful for users.

Reply:
Thanks for the suggestion! I added [a footnote](https://github.com/scalar-labs/scalardb/compare/bdad881...e466a16403e72fb8688bd0990db4dc2b001c5666) for that since the table would be complicated if I put the note in the table. 

@josh-wong Can you take a look at the diff when you get a chance? Thanks